test/DynamoCoreTests/DynamoViewModelUnitTest.cs

```diff
@@ -5,7 +5,7 @@
 using Dynamo.Models;
```

Comment:
Unit tests now rely on `DynamoShapeManager.Preloader` for its geometry preloading activities.